pkg/apis/sources/v1beta1/apiserver_lifecycle.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"knative.dev/pkg/apis"
+)
+
+const (
+	// ApiServerConditionReady has status True when the ApiServerSource is ready to send events.
+	ApiServerConditionReady = apis.ConditionReady
+
+	// ApiServerConditionSinkProvided has status True when the ApiServerSource has been configured with a sink target.
+	ApiServerConditionSinkProvided apis.ConditionType = "SinkProvided"
+
+	// ApiServerConditionDeployed has status True when the ApiServerSource has had it's deployment created.
+	ApiServerConditionDeployed apis.ConditionType = "Deployed"
+
+	// ApiServerConditionSufficientPermissions has status True when the ApiServerSource has sufficient permissions to access resources.
+	ApiServerConditionSufficientPermissions apis.ConditionType = "SufficientPermissions"
+
+	// ApiServerConditionEventTypeProvided has status True when the ApiServerSource has been configured with its event types.
+	ApiServerConditionEventTypeProvided apis.ConditionType = "EventTypesProvided"
```

Comment:
I think this should be removed. I think nobody is setting this anymore. I may have missed to remove it. 
Here and in the other versions if it is there

Reply:
done